usr/sbin/so-netsniff-ng-cron

```diff
@@ -3,32 +3,23 @@
 # Define initial variables
 SENSORTAB="/etc/nsm/sensortab"
 INTERFACES=$(grep -v '#' $SENSORTAB | awk '{print $1}')
-NETSNIFF_ENABLED="no"
-NETSNIFF_RUNNING="no"
 OS_DATE=$(date +"%Y-%m-%d")
 
 # Check to see if netsniff-ng is enabled.
 for i in $INTERFACES; do
 	if grep -q 'PCAP_ENABLED="yes"' /etc/nsm/$i/sensor.conf; then
-		NETSNIFF_ENABLED="yes"
+		# Check to see if netsniff-ng is running.
+        	if pgrep -af netsniff-ng > /dev/null 2>&1; then
+        		NETSNIFF_DATE=$(pgrep -af -oldest netsniff-ng | cut -d'/' -f6)
+        		if [ "$NETSNIFF_DATE" == $OS_DATE ]; then
```

Comment:
Could this be simplified by changing to `"$NETSNIFF_DATE" != $OS_DATE`, then replacing the `exit` with the `nsm_sensor_ps-restart`?  Could also then remove all instances of `else; exit`.